problemsets-mat223/modules/module7.tex

```diff
@@ -121,7 +136,27 @@
 \begin{example}
 	Find all vectors orthogonal to $\vec a=\mat{1\\1\\1}$ and $\vec b=\mat{1\\2\\1}$.
 
-	XXX Finish
+	To find all vectors orthogonal to $\vec a$ and $\vec b$, it is equivalent to solve
+	\[
+	\underbrace{\mat{1&1&1\\1&2&1}}_A\mat{x\\y\\z}=
+	\left[\begin{array}{c}\vec a\\\hline\vec b\end{array}\right]\vec x=
+	\mat{\vec a\cdot \vec x\\\vec b\cdot\vec x}
+	     =\mat{0\\0}.
+    \]
+    By row reduction on matrix A as follows:
```

Comment:
-> `Row reducing $A$, we get \[ \Rref(A) = ... , \] and so we get that the complete solution expressed in vector form is ...`